examples/scripts/script_depsi_processing.py

```diff
@@ -42,6 +42,8 @@ def get_free_port():
 # Parameters
 method = 'nmad'  # Method for selection
 threshold = 0.45  # Threshold for selection
+dist_thres = 200  # Distance threshold for network selection
+include_index = [101] # Force including the 101th point, use None if no point need to be included
```

Comment:
Can you extent this example to multiple points, so that the formatting is clear?
And would be good to include to nearby points here as well (e.g., 101 and 102, although they may actually be relatively far apart).